docs/cvl/expr.md

```diff
@@ -59,9 +58,19 @@ special_vars ::=
            | "lastMsgSig"
            | "_"
            | "max_uint" | "max_address" | "max_uint8" | ... | "max_uint256"
+           | "nativeBalances"
+		   
+cast_functions ::=
+    | require_functions | to_functions | assert_functions
 
-special_functions ::=
-           | "to_uint256" | "to_int256" | "to_mathint"
+require_functions ::=
+    | "require_uint8" | ... | "require_uint256" | "require_int8" | ... | "require_int256"
+
+to_functions ::=
```

Comment:
I thought we only have `to_mathint` now?

Reply:
I didn't try to clean up this whole grammar